src/V3Randomize.cpp

```diff
@@ -718,6 +718,37 @@ class ConstraintExprVisitor final : public VNVisitor {
             initTaskp->addStmtsp(methodp->makeStmt());
         }
     }
+    void visit(AstCountOnes* nodep) override {
+        // Convert it to (x & 1) + ((x & 2) >> 1) + ((x & 4) >> 2) + ...
```

Comment:
```suggestion
        // Convert it to (x & 1) + ((x & 2) >> 1) + ((x & 4) >> 2) + ...
        // TODO: Alternatively VL_COUNTONES_I's method, or ask solvers for new op
```

Reply:
I don't think we can use `VL_COUNTONES` here, because we need to convert the expression to SMT expression.

Reply:
I meant use the formula style it uses.  Just an idea, fine not to.